### PR TITLE
Tysen radovich assignment#3 This is the real one 

### PR DESCRIPTION
**Pull request by tyrado**

### CODE REVIEW

File: problems/kattis/different/kotlin/sleddog/src/main/main.kt [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-    while (line != null) {
-        var (a, b) = line.split(' ');
-        println(diff(a.toLong(), b.toLong()));
-        line = readLine();
-    }
-}
-
-fun diff(a: Long, b: Long): Long {
-    return Math.abs(a - b)
-}
```
Comment (sleddog) [inline problems/kattis/different/kotlin/sleddog/src/main/main.kt:14]: why were these files removed?

Comment (tyrado) [reply problems/kattis/different/kotlin/sleddog/src/main/main.kt:14]: @sleddog idk who those are I am only did the r2l.py
